evennia/contrib/game_systems/achievements/README.md

```diff
@@ -0,0 +1,119 @@
+# Achievements
+
+A simple, but reasonably comprehensive, system for tracking achievements. Achievements are defined using ordinary Python dicts, reminiscent of the core prototypes system, and while it's expected you'll use it only on Characters or Accounts, they can be tracked for any typeclassed object.
+
+## Installation
+
+Once you've defined your achievement dicts in a module, assign that module to the `ACHIEVEMENT_CONTRIB_MODULES` setting in your settings.py
+
+Optionally, you can specify what attribute achievement progress is stored in, with the setting `ACHIEVEMENT_CONTRIB_ATTRIBUTE`. By default it's just "achievements".
+
+There is also a command available to let players check their achievements - `from evennia.contrib.game_systems.achievements.achievements import CmdAchieve` and then add `CmdAchieve` to your default Character and/or Account cmdsets.
+
+#### Settings examples
+
+One module:
+
+```python
+# in server/conf/settings.py
+
+ACHIEVEMENT_CONTRIB_MODULES = "world.achievements"
+```
+
+Multiple modules, with a custom-defined attribute:
+
+```python
+# in server/conf/settings.py
+
+ACHIEVEMENT_CONTRIB_MODULES = ["world.crafting.achievements", "world.mobs.achievements"]
+ACHIEVEMENT_CONTRIB_ATTRIBUTE = "achieve_progress"
+```
+
+A custom-defined attribute including category:
+
+```python
+# in server/conf/settings.py
+
+ACHIEVEMENT_CONTRIB_MODULES = "world.achievements"
+ACHIEVEMENT_CONTRIB_ATTRIBUTE = ("achievements", "systems")
+```
+
+## Usage
+
+### Defining your achievements
+
+This achievement system is designed to use ordinary dicts for the achievement data - however, there are certain keys which, if present in the dict, define how the achievement is progressed or completed.
+
+- **name** (str): The searchable name for the achievement. Doesn't need to be unique.
+- **desc** (str): A longer description of the achievement. Common uses for this would be flavor text or hints on how to complete it.
+- **category** (str): The type of actions this achievement tracks. e.g. purchasing 10 apples might have a category of "buy", or killing 10 rats might have a category of "defeat".
+- **tracking** (str or list): The *specific* things this achievement tracks. e.g. the previous example of killing rats might have a `"tracking"` value of `"rat"`. An achievement can also track multiple things: instead of only tracking buying apples, you might want to track apples and pears. For that situation, you can assign it to a list of values to check against: e.g. `["apple", "pear"]`
```

Comment:
Reading this, it's not clear what are the valid values of these 'things' you talk about. What is an 'apple' or 'rat' here? Must they be matched to objects? Are they just lables?

A common achievement doesn't track anything - you just get it when you perform a particular action or fulfills some condition. How is that represented here? Is the 'tracking' field optional?